docs/installation.rst

```diff
@@ -69,3 +69,17 @@ Execute the following command to install the latest development build of Transfo
   NVTE_FRAMEWORK=jax pip install git+https://github.com/NVIDIA/TransformerEngine.git@main        # Build TE for JAX only.
   NVTE_FRAMEWORK=tensorflow pip install git+https://github.com/NVIDIA/TransformerEngine.git@main # Build TE for TensorFlow only.
   NVTE_FRAMEWORK=all pip install git+https://github.com/NVIDIA/TransformerEngine.git@main        # Build TE for all supported frameworks.
+  
+Installation (from source)
```

Comment:
Does the installation above continue to work? I'm not sure if it download the submodule automatically.